### PR TITLE
feature - login ui

### PR DESCRIPTION
**Pull request by at-lytruong**
add file
outlet for login, register, forgot

### CODE REVIEW

**Comment by at-daiho** [comment]
:+1: There are no linter violations.

File: MyApp/View/Controllers/Login/LoginViewController.swift [hunk 1]
```diff
@@ -10,7 +10,22 @@ import UIKit
 
 class LoginViewController: UIViewController {
 
+    @IBOutlet weak var userNameTextField: UITextField!
+
+    @IBOutlet weak var passwordTextField: UITextField!
+
```
Comment (at-hieutran) [inline MyApp/View/Controllers/Login/LoginViewController.swift]: IBOutlet miss private

File: MyApp/View/Controllers/Login/RegisterViewController.swift [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+import UIKit
+
+class RegisterViewController: UIViewController {
+
+    @IBOutlet weak var emailSignUpTextField: UITextField!
+
+    @IBOutlet weak var passwordSignUpTextField: UITextField!
+
+    @IBOutlet weak var nameSignUpTextField: UITextField!
+
```
Comment (at-hieutran) [inline MyApp/View/Controllers/Login/RegisterViewController.swift]: IBOutlet miss private

File: MyApp/View/Controllers/Login/LoginViewController.swift [hunk 3]
```diff
@@ -10,7 +10,22 @@ import UIKit
...
+
     override func viewDidLoad() {
         super.viewDidLoad()
     }
+
+    @IBAction func signInButtonTouchUpInSide(_ sender: Any) {
```
Comment (at-hieutran) [inline MyApp/View/Controllers/Login/LoginViewController.swift]: naming

File: MyApp/View/Controllers/Login/LoginViewController.swift [hunk 4]
```diff
@@ -10,7 +10,22 @@ import UIKit
...
+    }
+
+    @IBAction func forgotPassButtonTouchUpInSide(_ sender: Any) {
```
Comment (at-hieutran) [inline MyApp/View/Controllers/Login/LoginViewController.swift]: naming

File: MyApp/View/Controllers/Login/LoginViewController.swift [hunk 5]
```diff
@@ -10,7 +10,22 @@ import UIKit
...
+    }
+
+    @IBAction func signUpButtonTouchUpInSide(_ sender: Any) {
```
Comment (at-hieutran) [inline MyApp/View/Controllers/Login/LoginViewController.swift]: naming

File: MyApp/View/Controllers/Login/RegisterViewController.swift [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+    }
+
+    @IBAction func signUpButtonTouchUpInSide(_ sender: Any) {
```
Comment (at-hieutran) [inline MyApp/View/Controllers/Login/RegisterViewController.swift]: naming

File: MyApp/View/Controllers/Login/ForgotPassWordViewController.swift [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+
+    @IBOutlet private weak var resetEmailTextField: UITextField!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        view.backgroundColor = UIColor.black
+
+    }
+    @IBAction func resetPasswordButtonTouchUpInSide(_ sender: Any) {
```
Comment (at-hieutran) [inline MyApp/View/Controllers/Login/ForgotPassWordViewController.swift]: naming

**Comment by at-hieutran** [comment]
LGTM

**Pull request merged**